hasher-matcher-actioner/hmalib/lambdas/aws_secrets.py

```diff
@@ -0,0 +1,49 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+
+import boto3
+import base64
+import typing as t
+import os
+import ast
+from botocore.exceptions import ClientError
+
```

Comment:
I wasn't able to find a definitive answer on the tradeoffs, but many pieces of code I've seen simply create a resource at the top level, which under the hood has a client.

```
secrets = boto.resource("secretsmanager")
...
def foo():
  secrets.get_secret_value(val)
```

This can potentially save you some boilerplate

Reply:
Got it. I was following the python example here: https://console.aws.amazon.com/secretsmanager/home?region=us-east-1#!/secret?name=threatexchange%2Fapi_keys

But moving to top level makes sense